content/30.conclusions.md

```diff
@@ -1,27 +1,27 @@
 ## Conclusions
 
-A major challenge that we faced in carrying out this work is that it can be difficult to predict precise geographic origins for some groups of names.
+A major challenge that we faced in carrying out this work is to predict precise geographic origins for some groups of names.
 For example, we were unable to construct a classifier that could distinguish between names from Iberia (Spain and Portugal) and names from Spanish and Portuguese-speaking countries in Latin America.
 Discrepancies in representation between these groups are thus undetectable by our classifier.
 Hispanic honoree counts are influenced from Spain as well as Latin America.
-In these cases our analyses may substantially understate the extent to which minoritized scientists are underrepresented among honorees and authors.
+In these cases, our analyses may substantially understate the extent to which minoritized scientists are underrepresented among honorees and authors.
 
 Biases in authorship practices may also cause us to underestimate the composition of minoritized scientists within the field.
 We estimate the composition of the field using corresponding author status, but in neuroscience [@doi:10.1101/275362] and other disciplines [@doi:10.1371/journal.pbio.2004956] women are underrepresented among such authors.
 Such an effect would cause us to underestimate the number of women in the field.
 Though this effect has been studied with respect to gender, we are not aware of similar work examining race, ethnicity, or nationality.
 
-Measuring representation can lead to the questions about what the right level of representation is.
+An important questions to ask when measuring representation is what the right level of representation is.
 We suggest that considering equity may be more appropriate than strictly diversity.
-In addition to holding fewer corresponding authorship positions, female scientists are cited less often [@arxiv:2001.01002], invited by journals to submit papers less often [@doi:10.1371/journal.pbio.2004956] and suggested as reviewers less often [@doi:10.1038/541455a].
+In addition to holding fewer corresponding authorship positions, on average, female scientists of different disciplines are cited less often [@arxiv:2001.01002, @doi:10.1002/ece3.4993], invited by journals to submit papers less often [@doi:10.1371/journal.pbio.2004956], suggested as reviewers less often [@doi:10.1038/541455a], and receive significantly lower review scores [@doi:10.1002/ece3.4993].
```

Comment:
```suggestion
In addition to holding fewer corresponding authorship positions, on average, female scientists of different disciplines are cited less often [@arxiv:2001.01002, @doi:10.1002/ece3.4993], invited by journals to submit papers less often [@doi:10.1371/journal.pbio.2004956], suggested as reviewers less often [@doi:10.1038/541455a], and receive significantly worse review scores [@doi:10.1002/ece3.4993].
```
Since it's about papers, I switched to "worse" from "lower"